services/news/src/views/index.ejs

```diff
@@ -60,6 +66,74 @@
     <hr>
     <p><a href=<%= `https://${HOME_HOST}:${EXTERNAL_PORT}` %>>back to top</a>
   </footer>
+  <script>
+    function addScriptEl(src, className, resolve) {
+      const scriptEl = document.createElement('script')
+      scriptEl.src = src
+      scriptEl.className = className
+
+      if (resolve) {
+        scriptEl.addEventListener('load', resolve)
+      }
+
+      document.body.appendChild(scriptEl)
+      return scriptEl
+    }
+
+    async function startSequentialAuction(adUnit, sellers) {
```

Comment:
Maybe add a link as a comment to our pa-webinar-2 deck, which explains the term "sequentialAuction"? I guess not many people will be familiar with that term.

Reply:
Super duper important call. I've added a summary of the auction and linked the resources people can use to learn more.